.github/workflows/helm-docs.yaml

```
@@ -0,0 +1,164 @@
+name: "Helm Docs Check"
+
+on:
+  pull_request:
+    paths:
+      - "**/*/values.yaml"
+      - "**/*/*/values.yaml"
```

If you want the action be triggered once the PR is updated you need to add the `types: [opened, synchronize, reopened]` as a trigger. Otherwise it will run only once when the PR is opened
```suggestion
  pull_request:
    types: [opened, synchronize, reopened]
    paths:
      - "**/*/values.yaml"
      - "**/*/*/values.yaml"
```

---

thank you for the pointer. Fixed in e149e6eb5b38a27ff1246bd0ba852e48cc7ea754